content/applications/inventory_and_mrp/inventory/warehouses_storage/inventory_management/create_a_second_warehouse.rst

```diff
@@ -2,99 +2,135 @@
+- :guilabel:`Resupply Subcontractors`: allow subcontractors to be resupplied with goods from this
+  warehouse.
+- :guilabel:`Manufacture to Resupply`: when products are manufactured, they can be manufactured in
+  this warehouse.
+- :guilabel:`Manufacture`: the route that should be followed when manufacturing goods inside
+  the warehouse. Choose from one, two, and three-step production.
+- :guilabel:`Buy to Resupply`: when products are purchased, they can be delivered to this warehouse.
+- :guilabel:`Resupply From`: tick the checkbox next to the warehouse that this warehouse should be
+  resupplied from. Resupply routes are automatically created for the selected warehouse.
+
+.. image:: create_a_second_warehouse/create-a-second-warehouse-warehouse-configuration-tab.png
+   :align: center
+   :alt: Filled out Warehouse Configuration tab settings on warehouse creation form.
 
-After filling out the form, click :guilabel:`Save` and the new warehouse will be created.
+Once ready, click :guilabel:`Save` to save the new warehouse's settings.
 
 Add inventory to a new warehouse
 ================================
 
-If a new warehouse is created that has existing inventory in it, the inventory counts should be
-added to Odoo so that the stock listed in the Odoo database reflects what is in the physical
-warehouse. To add inventory to a new warehouse, navigate to :menuselection:`Inventory -->
-Operations --> Inventory Adjustments`, and then click :guilabel:`Create`. The inventory adjustment
-form can then be filled out as follows:
-
-- :guilabel:`Inventory Reference`: the name or code that the inventory adjustment can be referred to
-  by
-- :guilabel:`Locations`: the location(s) where the inventory is stored; include the new warehouse
-  and any locations within it that inventory will be added to
-- :guilabel:`Products`: include all products that will be added to inventory or leave blank to
-  select any product during the next step
-- :guilabel:`Include Exhausted Products`: include products with a quantity of zero; does not affect
-  inventory adjustments for new warehouses since they have no existing inventory
-- :guilabel:`Accounting Date`: the date used by accounting teams for bookkeeping related to the
-  inventory
-- :guilabel:`Company`: the company that owns the inventory; can be set as the user's company or as a
-  customer or vendor
-- :guilabel:`Counted Quantities`: choose whether the counted quantities for products being added
-  should default to stock on hand or zero; does not affect inventory adjustments for new warehouses
-  since they have no existing inventory
-
-.. image:: create_a_second_warehouse/inventory-adjustment-configuration.png
-   :align: center
-   :alt: A filled out form for an inventory adjustment.
+If a new warehouse is created with existing inventory in it, the inventory count should be added to
+Odoo, so the stock listed in the Odoo database reflects the real stock count in the physical
+warehouse.
```

Comment:
The current explanation does a good job of explaining what we need to do in Odoo, but we can reword it to elaborate on *why* we need to adjust the stock in the first place.

```suggestion
If a new warehouse is created to represent a real-world storage location with existing stock, update the warehouse's stock using an inventory adjustment.
```

Reply:
Suggestion implemented ✅